hal/targets/hal/TARGET_NORDIC/TARGET_NRF5/LF_Clock_config.md

```diff
@@ -0,0 +1,96 @@
+
+## LF Clock configuration using mbed configuration system
+
+In order to provide the configuration for a low frequency (LF) clock, add a description of the LF clock inside a *.json file, for example in mbed_json.json.
```

Comment:
It is not inside any json file.
Please detail for the target configuration (hal/targets.json) and application (mbed_app.json).